outgoingra/src/main/java/no/bankid/outgoing/ra/RaRequirements.java

```diff
@@ -6,139 +6,319 @@
 import javax.ws.rs.core.Response;
 
+import static no.bankid.outgoing.ra.HttpSignatureHeaders.DATE;
+import static no.bankid.outgoing.ra.HttpSignatureHeaders.DIGEST;
+import static no.bankid.outgoing.ra.HttpSignatureHeaders.SIGNATURE;
+
 @OpenAPIDefinition(
         info = @Info(
-                title = "Integration of BankID app with bank's BankID RA service",
+                title = "Integration of BankID App with bank's BankID RA service",
                 version = "1.2",
-                description = "Defines methods needed to be implemented by Registration Authority service in banks " +
-                        "wanting to add BankID app as an OTP mechanism for their BankID Netcentric users.\n\n" +
-                        "If an RA service implements these methods, than the activation of an BankID app will use " +
-                        "them to connect the BankID app with the endUser's BankID."
-        //      ,contact =
-        //      @Contact(url = "http://gigantic-server.com", name = "Fred", email = "Fred@gigagantic-server.com")
-        )
+                description = "Defines methods needed to be implemented by a Registration Authority service in banks " +
+                        "wanting to add BankID App as an OTP mechanism for their BankID Netcentric users.<p>" +
+                        "If an RA service implements these methods, than the activation of an BankID App will use " +
+                        "them to connect the BankID App with the endUser's BankID.</p>"
+        ),
+        tags = {
+                @Tag(name = "Basic RA Requirements",
+                        description = "Adds or deletes BankID App from an endUser's BankID"),
+                @Tag(name = "Activation without Code Device",
+                        description = "Activation of BankID App without no other Code Device")
+
+        },
+        servers = {
+                @Server(description = "Preprod Ra-lite",
+                        url = "https://ra-preprod.bankidnorge.no/api/enduser/bankid/netcentric/vipps/bapp")
+        }
+
 )
 
-@Tag(name = "RaRequirements",
-        description = "Administrate (BankID app OTP, Netcentric BankID) relation in endUser's bank's RA")
-@Path("/external/ra/")
-@Produces({MediaType.APPLICATION_JSON})
 
+@Path("/")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
 public interface RaRequirements {
-    @Operation(description = "Adds BankID app to an endUser's BankID OTP mechanisms in a given bank")
+
+    String DESCRIPTION_SIGNATURE = "The Signature element, as described " +
+            "in <a href=\"https://tools.ietf.org/html/draft-cavage-http-signatures-12\">Internet-Draft - Signing HTTP Messages</a>.";
+    String EXAMPLE_SIGNATURE = "keyId=\"fa998090\",algorithm=\"rsa-sha256\"," +
+            "headers=\"(request-target) date x-client-clientname x-dataownerorgid " +
+            "x-client-requestid x-customerid\",signature=\"o7zK892....\"";
```

Comment:
Is this documentation string still correct?

Haven't we removed x-client-requestid etc...

Reply:
Thank you for seeing that. Fixed.